### PR TITLE
First GUI

### PR DESCRIPTION
**Pull request by guillemglez**
UI added.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1,9 +1,49 @@
...
+  position: absolute;
+  text-align: right;
+  width:99%;
+
+  padding-right: 10px;
+	padding-bottom: 10px;
+  padding-top: 10px;
+}
+
+/*#navbox {
```
Comment (ecaruyer) [inline css/main.css]: consider removing these comments

File: gui/functions.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+function moveCameraXY() {
+  if (camera.position.z == phantom.radius()*1.5) {
+    camera.position.set(0, 0, 0);
```
Comment (ecaruyer) [inline gui/functions.js]: You may also want to call camera.up(0, 1, 0)

File: gui/left.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+PHANTOM
```
Comment (ecaruyer) [inline gui/left.html]: I think you can remove thoses two files now.

File: js/animate.js [hunk 4]
```diff
@@ -1,19 +1,31 @@
...
 
 function init() {
+  request = new XMLHttpRequest();
+  request.overrideMimeType("text/plain");
+  request.open("get", path, true);
+  request.onreadystatechange = function() {
+    if ( (request.readyState === 4) && (request.status === 200) ) {
+      show();
+    }
+  }
```
Comment (ecaruyer) [inline js/animate.js]: missing semicolon here?

File: js/animate.js [hunk 5]
```diff
@@ -46,14 +58,23 @@ function init() {
...
   renderer.render(scene, camera);
 
   // KeyPress consecution loop for features show.
+
+  // Add mouse control to the camera
+  controls = new THREE.TrackballControls( camera );
+  controls.enableZoom = true;
+  controls.rotateSpeed = 2.5;
+  controls.zoomSpeed = 1;
+  controls.noPan=true;
```
Comment (ecaruyer) [inline js/animate.js]: space before and after =

File: gui/functions.js (hunk 2 above)
Comment (guillemglez) [reply gui/functions.js]: Camera.lookAt(0,0,0) is already enabled when the camera is placed in the scene, so just moving the camera is needed.

**Pull request merged**